### ISSUE TITLE
Text Classification - Loss function

### ISSUE THREAD

**Issue by bokstaller** [flairNLP/flair#2568]
Hi everyone,

just a short question regarding loss functions. Is it possible to change the loss function when training the text classifier with flair? I would need to use the triplet loss.

Thanks in advance,

Jonas.

**Comment by whoisjones** [comment]
Duplicate of #2567
